konfera/models/organizer.py

```diff
@@ -0,0 +1,24 @@
+from konfera.models.countries import COUNTRIES
+from django.db import models
+from konfera.models.abstract import KonferaModel
+
+
+class Organizer(KonferaModel):
+    COUNTRY_DEFAULT = 'SK'
+
+    title = models.CharField(max_length=128)
+    street = models.CharField(max_length=128)
+    street2 = models.CharField(max_length=128)
+    city = models.CharField(max_length=128)
+    postcode = models.CharField(max_length=12)
+    country = models.CharField(
```

Comment:
There are also company/organization details like ID, tax ID or VAT ID so I suppose we want to keep all the address details.